src/BCTBondV1.ts

```diff
@@ -1,6 +1,6 @@
 import { BigDecimal } from '@graphprotocol/graph-ts'
 
-import { BondCreated, BondRedeemed } from '../generated/BCTBondV1/BCTBondV1'
+import { BondCreated, BondRedeemed } from '../generated/BondV1/BondV1'
```

Comment:
I am unable to build this currently as the generated/BondV1/BondV1 path does not exist. Each datasource should be referencing the import within itself, so for this file the import path would be `../generated/BCTBondV1/BondV1`

This needs to be updated on all other source files as well.